### PR TITLE
Add apt-cache

### PR DESCRIPTION
**Pull request by sbrl**
I use the `apt-cache` command quite a bit and it's rather useful, so I thought that I'd something here since it's a non obvious command.

### CODE REVIEW

File: pages/linux/apt-cache.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# apt-cache
+
+> Debian and Ubuntu package query tool
+
+ - Search for a package in your current sources
+
+```
+apt-cache search {{query}}
```
Comment (igorshubovych) [inline pages/linux/apt-cache.md]: See CONTRIBUTING.md for Markdown format.

We use 

```Markdown
- Example 1

`command args`
```

**Comment by igorshubovych** [comment]
The page looks good. Please fix the format.

**Comment by waldyrious** [comment]
LGTM, apart from @igorshubovych's comments.

**Comment by sbrl** [comment]
@igorshubovych Done and fixed. Should I squash the commit too?

**Comment by felixonmars** [comment]
Yes, please always do :)

**Comment by sbrl** [comment]
@felixonmars All squashed and ready to go!

I'll do another simple man page for imagemagick's montage command after this PR I think.

**Comment by igorshubovych** [comment]
Thank you for contribution. Please use feature (topic) branches in future. This way you can work on several pages simultaneously.

**Comment by sbrl** [comment]
@igorshubovych What's a feature (topic) branch? I haven't heard of one of those before.

**Comment by igorshubovych** [comment]
Read here about feature branches:
https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md#submitting-a-pull-request

**Comment by igorshubovych** [comment]
Also take a note that you broke the build:
https://github.com/tldr-pages/tldr/commit/c7f5c3440206a0bc32f7fdd52768ada09cd43a15

**Comment by sbrl** [comment]
@igorshubovych Oops! I didn't realise. Should I open another PR to fix that?

And a feature branch is just a branch with a name that reflects the feature (in this case tldr page) that you are adding?

**Comment by igorshubovych** [comment]
NP. I fixed it already.

But please follow the instructions on CONTRIBUTING.md next time.

**Comment by sbrl** [comment]
@igorshubovych Will do! I'll take a look.

**Pull request merged**